lib/adal.js

```diff
@@ -1015,10 +1112,18 @@ var AuthenticationContext = (function () {
         return crackedToken;
     };
 
+    /**
+     * Converts string to represent binary data in ASCII string format by translating it into a radix-64 representation and returns it
+     * @ignore
+     */
     AuthenticationContext.prototype._convertUrlSafeToRegularBase64EncodedString = function (str) {
         return str.replace('-', '+').replace('_', '/');
     };
 
+    /**
+     * Generates the parameters for the authorization endpoint URL.
```

Comment:
Serializes the parameters for the authorization endpoint URL and returns the serialized uri string.